File: arm-fragment/core/src/main/java/com/iur/arm/fragment/core/anim/DefaultHorizontalAnimator.java
package com.iur.arm.fragment.core.anim;

import android.os.Parcel;
import android.os.Parcelable;

import com.iur.arm.fragment.core.R;


public class DefaultHorizontalAnimator extends FragmentAnimator implements Parcelable {

    public DefaultHorizontalAnimator() {
        enter = R.anim.h_fragment_enter;
        exit = R.anim.h_fragment_exit;
        popEnter = R.anim.h_fragment_pop_enter;
        popExit = R.anim.h_fragment_pop_exit;
    }

    protected DefaultHorizontalAnimator(Parcel in) {
        super(in);
    }

    @Override
    public void writeToParcel(Parcel dest, int flags) {
        super.writeToParcel(dest, flags);
    }

    @Override
    public int describeContents() {
        return 0;
    }

    public static final Creator<DefaultHorizontalAnimator> CREATOR = new Creator<>() {
        @Override
        public DefaultHorizontalAnimator createFromParcel(Parcel in) {
            return new DefaultHorizontalAnimator(in);
        }

        @Override
        public DefaultHorizontalAnimator[] newArray(int size) {
            return new DefaultHorizontalAnimator[size];
        }
    };
}

File: arm-fragment/core/src/main/java/com/iur/arm/fragment/core/anim/DefaultHorizontalAnimator.kt
package com.iur.arm.fragment.core.anim

import com.iur.arm.fragment.core.R

class DefaultHorizontalAnimator : FragmentAnimator() {
    init {
        enter = R.anim.h_fragment_enter
        exit = R.anim.h_fragment_exit
        popEnter = R.anim.h_fragment_pop_enter
        popExit = R.anim.h_fragment_pop_exit
    }
}
